### PR TITLE
Add port param

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@vogdb please review

File: bin/janus-cluster [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+var port = argv.port || 8080;
+
 var Cluster = require('../lib/Cluster');
 var HttpServer = require('../lib/HttpServer');
 
 var cluster = new Cluster();
-var httpServer = new HttpServer(8080, cluster);
+var httpServer = new HttpServer(port, cluster);
 httpServer.start();
+console.log('Started http server at port ' + port);
```
Comment (vogdb) [inline bin/janus-cluster]: console.log

Comment (tomaszdurka) [inline bin/janus-cluster]: As discussed looks good.

**Comment by vogdb** [comment]
as discussed `console.log` is a temporary logger here.

lgtm

**Pull request merged**